wbooks/src/app/screens/BookDetails/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { View } from 'react-native';
+import ScreenWrapper from '@components/ScreenWrapper';
+
+function BookDetails() {
+  return (
+    <ScreenWrapper>
+      <View />
+    </ScreenWrapper>
```

Comment:
No, this is not ok. You only need the `SafeAreaView` here, not the `StatusBar`. The RN App only need one `StatusBar` per App

Reply:
Good to know, refactored